.github/workflows/main_ci.yml

```diff
@@ -90,5 +90,5 @@ jobs:
       - name: codecov check
         uses: codecov/codecov-action@v2
         with:
-          files: coverage.xml
+          files: $RUNNER_TEMP/coverage.xml,/home/runner/work/_temp/coverage.xml,/Users/runner/work/_temp/coverage.xml
```

Comment:
@thomasrockhu Any idea why I have to specify the absolute paths here instead of `codecov` action being able to expand the `$RUNNER_TEMP` variable? For now, I'll just use this, but would be more maintainable if we could use the first path in this list only.

Reply:
yeah, that's pretty confusing to me. Are you able to run in verbose mode? I think we'd get a better understanding there.